### PR TITLE
Rubocop changes

### PR DESCRIPTION
**Pull request by ananyo2012**
@sonalkr132 please review this PR for rubocop changes.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+inherit_from: .rubocop_todo.yml
+AllCops:
+  Include:
+    - '**/Rakefile'
+    - '**/config.ru'
+  Exclude:
+    - 'db/schema.rb'
+    - 'config/**/*'
+    - 'bin/**/*'
```
Comment (sonalkr132) [inline .rubocop.yml:9]: exclude vendor files as well. That is the reason your test is failing.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Offense count: 1
+Metrics/CyclomaticComplexity:
+  Max: 12
+
+# Offense count: 108
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 148
+
```
Comment (sonalkr132) [inline .rubocop_todo.yml]: Please don't use such long length line. 100 should be more than enough.

Comment (ananyo2012) [inline .rubocop_todo.yml]: but some lines are more than 100.what to do with them?

Comment (sonalkr132) [inline .rubocop_todo.yml]: You need to break them, align them in such a way that they don't get wider than 100 characters. We need to customize our code that it passes rubocop and not customize rubocop that our code passes.

I think I have mentioned this [ruby-style-guide](https://github.com/bbatsov/ruby-style-guide). Check out the section where they talk about line length. You can also have a look at these:
- http://stackoverflow.com/questions/10522414/breaking-up-long-strings-on-multiple-lines-in-ruby-without-stripping-newlines
- http://stackoverflow.com/questions/12083011/how-to-break-up-long-lines-of-ruby
- http://codereview.stackexchange.com/questions/36782/how-do-i-refactor-lines-of-ruby-code-that-run-too-long-due-to-method-chaining-or

If you come across something which you can't your head around then you should let me know.

**Comment by ananyo2012** [comment]
@sonalkr132 i have made the changes please review the PR.

File: app/controllers/like_counts_controller.rb [hunk 3]
```diff
@@ -28,26 +28,39 @@ def create
...
+        format.html { redirect_to(action: 'index', controller: 'new_threads') }
         format.js
         format.json { render :show, status: :created, location: @like_count }
-        # redirect_to(:action => 'show', :controller => 'user', :user_id => @user.user_id)
+        # redirect_to(
+        #   :action => 'show',
+        #   :controller => 'user',
+        #    :user_id => @user.user_id
+        # )
       else
```
Comment (sonalkr132) [inline app/controllers/like_counts_controller.rb:45]: please remove useless commented codes, like two cases above from all the controller files. You don't need to edit commit for it, just add an extra commit.

Comment (ananyo2012) [inline app/controllers/like_counts_controller.rb:45]: ok i will do it

Comment (ananyo2012) [inline app/controllers/like_counts_controller.rb:45]: do i need to remove comments that are generated by default like in spec helper, uploader file of cloudinay etc. ?

Comment (sonalkr132) [inline app/controllers/like_counts_controller.rb:45]: no just controller files and only one which look like above two.

**Comment by ananyo2012** [comment]
@sonalkr132 please review the commit that i made.

**Pull request merged**